spec/requests/users_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe "Users", type: :request do
+    let(:valid_attributes) {
+        {email: 'john@example.com', password: 'test_password', password_confirmation: 'test_password'} 
+    }
+
+    let(:invalid_attributes) {
+        {email: 'john@example.com', password: 'test_password', password_confirmation: 'unmatching_test_password'} 
+    }
+
+    describe "GET /new" do
+        it "renders a successful response" do
+        get new_user_url
+        expect(response).to be_successful
```

Comment:
What is the difference between these two? Are both required to test the response?